src/k8s/pkg/k8sd/pki/control_plane.go

```diff
@@ -34,6 +36,7 @@ type ControlPlanePKIOpts struct {
 	Hostname                  string
 	DNSSANs                   []string
 	IPSANs                    []net.IP
+	ExtraSANs                 string
```

Comment:
we already have fields for DNSSans and IPSans. do not add an extra field, rather update the DNSSans and IPSans accordingly